static/js/context/auth.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, FC, ReactNode, useContext } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+
+const checkAuth = async () =>
+  await fetch("/1.0/certificates").then((response) => response.status !== 403);
+
+interface ContextProps {
+  isAuthenticated: boolean;
+  isAuthLoading: boolean;
+}
+
+const initialState: ContextProps = {
+  isAuthenticated: false,
+  isAuthLoading: true,
+};
+
+export const AuthContext = createContext<ContextProps>(initialState);
+
+interface ProviderProps {
+  children: ReactNode;
+}
+
+export const AuthProvider: FC<ProviderProps> = ({ children }) => {
```

Comment:
Could you catch errors and show them in the console at least?

Reply:
Currently we don't have a practice of logging errors to the console in this project.